app/assets/stylesheets/admin.scss

```diff
@@ -22,6 +22,17 @@ label {
     padding: 8px;
   }
 }
+
+/****
+The admin area's shared <main> has padding of var(--su-2).
+In the new member index view we want inner content to be able to take up full screen width.
+Since we will want to be able to do the same in later admin area redesigns, we have created a helper class,
+which may be more easily removed once we no longer wish admin layouts to have a padded <main> by default. 
+****/
+.overflow-admin-main-layout-padding {
+  margin: calc(-1 * var(--su-2));
+}
+
```

Comment:
I can't decide if this is the right approach or not 😅  any thoughts?

We have two main options:
1. Use a class like this to make the new views the "exception" to the rule, and remove it later when the exceptions begin to outnumber the other pages
2. Remove the padding from the admin `<main>` element, and add it manually in every page other than the member index view (or I guess some pages maybe look acceptable without it 🤷‍♀️ )


Option 1. is based on the assumption that as we update admin sections, we'll be repeating this pattern of small-screen layouts featuring a full-screen-width card for content.

A third option is to add negative margins directly on the admin member index view e.g. `-m-2`), but I'd rather avoid that as 
- without some kind of documentation it seems weird
- if my assumption above is correct, we'll end up with random `-m-2` peppered in various places, and either neglect to update the overall layout, or have a harder job rooting things out later

Reply:
It works as expected in the UI 🚀 

In terms of implementation, I support Option 1 more.

Reply:
I'm in favor of option 1 as well! 

In the event that we do not update the Admin sections with this pattern, we won't have to update code in _too_ many places since it'll be encapsulated in the class and used only within the Member Index View (for now), hopefully making it pretty straightforward to pivot/overhaul.

Reply:
Awesome, thanks you two 🌟